config/template.js

```diff
@@ -39,7 +39,7 @@ const SCHEMA_TEMPLATE = Joi.object()
         maintainer: TEMPLATE_MAINTAINER,
         config: Job.job
     })
-    .requiredKeys('name', 'version', 'description', 'maintainer', 'config.steps');
+    .requiredKeys('name', 'version', 'description', 'maintainer', 'config', 'config.steps');
```

Comment:
👍  
can you also add config.image? to address https://github.com/screwdriver-cd/screwdriver/issues/510